bench/internals.rs

```diff
@@ -0,0 +1,206 @@
+#![deny(warnings)]
+
+use felt::{felt_str, Felt};
+use lazy_static::lazy_static;
+use num_traits::Zero;
+use starknet_rs::{
+    business_logic::{
+        fact_state::in_memory_state_reader::InMemoryStateReader,
+        state::{cached_state::CachedState, state_api::State},
+        transaction::objects::{
+            internal_declare::InternalDeclare, internal_deploy::InternalDeploy,
+            internal_deploy_account::InternalDeployAccount,
+            internal_invoke_function::InternalInvokeFunction,
+        },
+    },
+    core::contract_address::starknet_contract_address::compute_class_hash,
+    definitions::general_config::StarknetChainId,
+    public::abi::VALIDATE_ENTRY_POINT_SELECTOR,
+    services::api::contract_class::ContractClass,
+    utils::{felt_to_hash, Address},
+};
+use std::{hint::black_box, path::PathBuf};
+
+lazy_static! {
+    // include_str! doesn't seem to work in CI
+    static ref CONTRACT_CLASS: ContractClass = ContractClass::try_from(PathBuf::from(
+        "starknet_programs/account_without_validation.json",
+    ))
+    .unwrap();
```

Comment:
Is there any way to build the `ContractClass` from bytes? In that case you could use `include_bytes!` instead of creating a path that needs to be read from.

Reply:
Done!

Reply:
The GH workflow that runs clippy doesn't install deps and so cannot compile the cairo programs. I'll revert this change.